start

```diff
@@ -49,6 +52,18 @@ function main() {
     echo "--enable-core-artificially-accelerate-time-for-testing is only supported without captive core" >&2
     exit 1
   fi
+  if [ "$ENABLE_LOGS" = "true" ]; then
+    print_service_logs &
+    start \
+      >  >(sed "s/^/${cyan}quickstart   | $clear/") \
+      2> >(sed "s/^/${cyan}quickstart   | $clear/" >&2)
```

Comment:
TODO: These redirects don't gel well with the redirects on the exec supervisord that is within start. I'll look into this more.

Reply:
Fixed in https://github.com/stellar/quickstart/pull/413/commits/678ed9585203337fda6d420c6ac23935f45e2d2e